developer_guide/content/sections/sqs.md

```diff
@@ -0,0 +1,29 @@
+
+# Establish connection with your SQS 
+
+* be sure your user have proper permissions
+
+```ruby
+    aws_access_key_id =  'AxxxxxxxxxxxxxxxA'
```

Comment:
I don't want to encourage hard-coded credentials at all. Given that we have a default credential provider chain explained in the guide already, I'd stick with:

```ruby
Aws::SQS::Client.new(region: region_name)
```